src/Splat.Microsoft.Extensions.DependencyInjection/README.md

```diff
@@ -0,0 +1,103 @@
+# Splat.Microsoft.Extensions.DependencyInjection
+
+## Using Autofac
+
+`Splat.Microsoft.Extensions.DependencyInjection` is an adapter for `IMutableDependencyResolver`.
+It allows you to register your application dependencies in a MS DI `Container`.  You can then use the container as Splat's internal dependency resolver.
+You can also choose to have the conainer controlled externally, for example using a [Generic Host](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/host/generic-host?view=aspnetcore-2.2#configureappconfiguration).
+
+### Register the Container
+
+```cs
+// call this method from your apps constructor or early initialization code
+
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging.EventLog;
+using ReactiveUI;
+using Splat;
+using Splat.Microsoft.Extensions.DependencyInjection;
+using Splat.Microsoft.Extensions.Logging;
+using System;
+using System.Linq;
+// using Windows.UI.Xaml
+
+sealed partial class App // : Application
+{   
+  public App()
+  {
+    Init();
+    /* Some other initialization stuff */
+  }                     
+
+  public IServiceProvider Container { get; private set; }
+
+  void Init()
+  {
+    var host = Host
+      .CreateDefaultBuilder()
+      .ConfigureServices(services =>
+      {
+        services.UseMicrosoftDependencyResolver();
+        var resolver = Locator.CurrentMutable;
+        resolver.InitializeSplat();
+        resolver.InitializeReactiveUI();
+
+        // Configure our local services and access the host configuration
+        ConfigureServices(services);
+      })
+      .ConfigureLogging(loggingBuilder =>
+      {
+        /*
```

Comment:
Remove code comments?

Reply:
Should we keep them there? In a UWP app, it causes a lot of $hit, took me time to nail.

Reply:
Fair enough.